android/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <uses-permission android:name="com.android.vending.BILLING"/>
 
     <application
+        android:name="com.example.app.Application"
```

Comment:
Just as a reminder, changing the application package may influence in the tests, due to it not being the same as the google play one with the products

Reply:
That's not entirely true.
With this code i am still able to do the subscriptions and cancel them.

Maybe in some other situation this might affect but not so far!

Reply:
Ok, if it is still working maybe that depends only on the gradle application name